python/dgl/data/qm9.py

```diff
@@ -0,0 +1,127 @@
+"""QM9 dataset for graph property prediction (regression)."""
+import os
+import numpy as np
+import scipy.sparse as sp
+import torch
+
+from .dgl_dataset import DGLDataset
+from .utils import download
+from ..convert import graph as dgl_graph
+from ..transform import to_bidirected
+
+class QM9Dataset(DGLDataset):
+    r"""QM9 dataset for graph property prediction (regression)
+
+    This dataset consists of 13,0831 molecules with 12 regression targets.
+    - id: [0, 1, ..., 133884], a unique identifier for each molecule.
```

Comment:
Do we expect users to directly access `id`, `R`, `Z`, `N`? If so, we should tell users how to access them. If not, we can simply remove them and just explain the returned results in `__getitem__`.

Reply:
Actually, I think `id` is useless which could be removed. And the second way may be better, I'll add some comments in `__getitem__`